### PR TITLE
feat(config): add new 700 series Jasco devices

### PR DESCRIPTION
**Pull request by AlCalzone**

### CODE REVIEW

File: packages/config/config/devices/0x0063/59341_zwa3017.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	"manufacturer": "Enbrighten",
+	"manufacturerId": "0x0063",
+	"label": "59341 / ZWA3017",
+	"description": "In-Wall Toggle Dimmer, QFSW, 700S",
+	"devices": [
+		{
+			"productType": "0x4944",
+			"productId": "0x3431"
+		}
+	],
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0063/59341_zwa3017.json]: This file has the same IDs as 59335_zwa3017.json. We need to merge them into one.

Comment (AlCalzone) [reply packages/config/config/devices/0x0063/59341_zwa3017.json]: @jascoproducts

Comment (jascoproducts) [reply packages/config/config/devices/0x0063/59341_zwa3017.json]: > This file has the same IDs as 59335_zwa3017.json. We need to merge them into one.

Yes, let's merge.

On 59341_zwa3017.json:
- Change the label to "59341 / 59335 / ZWA3017"
- Change the filename to "59341_59335_zwa3017"

Description will remain the same.

Go ahead and remove 59335_zwa3017.json.

**Review by jascoproducts — DISMISSED** [review]

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
@zwave-js-bot approve

**Pull request merged**